scripts/fetch_validation_data.py

```diff
@@ -39,33 +40,82 @@
 """
 
 
+def single_result_formatter(row):
+    return row[0]
+
+
+def write_results(filename, formatter=single_result_formatter):
+    with open(filename, 'w') as f:
+        for row in cursor:
+            f.write(formatter(row))
+            f.write('\n')
+
+
 def fetch_data_for_date(date):
     date_string = date.strftime(date_format)
     print 'Fetching for {date}'.format(date=date_string)
     cursor.execute(QUERY_FORMAT, [date_string + '%'])
-    with open('validations/{date}.txt'.format(date=date_string), 'w') as f:
-        for row in cursor:
-            f.write(row[0])
-            f.write('\n')
+    write_results('validations/{date}.txt'.format(date=date_string))
 
 
 def fetch_unlisted_addon_ids():
     print 'Fetching unlisted addons'
     cursor.execute('SELECT guid FROM addons WHERE is_listed=0 '
                    'AND guid IS NOT NULL;')
-    with open('validations/unlisted-addons.txt', 'w') as f:
-        for row in cursor:
-            f.write(row[0])
-            f.write('\n')
+    write_results('validations/unlisted-addons.txt')
+
+
+def fetch_lite_addon_ids():
+    print 'Fetching STATUS_LITE addons'
+    cursor.execute('SELECT guid FROM addons WHERE status=8 '
+                   'AND guid IS NOT NULL;')
+    write_results('validations/lite-addons.txt')
+
+
+def fetch_validations():
+    today = datetime.today()
+    for i in range(30, 0, -1):
+        date = today - timedelta(days=i)
+        fetch_data_for_date(date)
+
+
+def fetch_manual_reviews():
+    def formatter(row):
+        return json.dumps(
+            {'guid': row[0], 'version': row[1], 'action': row[2],
+             'created': row[3].strftime('%Y-%m-%dT%H:%M%S')})
+
+    query = """
+        SELECT a.guid, v.version, la.action, la.created
+        FROM versions v
+        JOIN addons a on a.id=v.addon_id
+        JOIN log_activity_version lav ON lav.version_id=v.id
+        JOIN log_activity la ON la.id=lav.activity_log_id
+        WHERE a.guid IS NOT NULL AND a.is_listed=0 AND
+              la.action IN (42, 43) -- (PRELIMINARY_VERSION, REJECT_VERSION)
+        ;"""
```

Comment:
I figured they wouldn't have a review entry, looks like they do. Good catch, I'll update it.